_posts/2016/2016-05-26-magdeburger-developer-days.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Magdeburger Developer Days"
+date: "2016-05-26 12:30:20"
+image: blog-header/magdeburg-dev.jpg
+categories: events
+authors: ["Bastian and Benjamin"]
+---
+
+On May, 18th 2016, the first [Developer Days](http://md-devdays.de/) took place in the exhibition halls of Magdeburg, Germany.
+Although this Java and .NET developer conference was organised for the first time, many developers from across Germany participated and followed the invitation from the [Twitter](https://twitter.com/MiB_MD_DevDays) channel of Michael Blume, the initiator and organiser.
+
+{% image blog/blog-magdeburg-1.jpg %}
+
+## The setting
+
+The exhibition hall was parted into four large rooms, named after famous pioneers of the Information Technologies, like [Konrad Zuse](https://en.wikipedia.org/wiki/Konrad_Zuse) and [Bjarne Stroustrup](https://en.wikipedia.org/wiki/Bjarne_Stroustrup).
+The lobby offered bar tables and an open space that could consistently be used for interesting discussions about the latest technologies, side projects and networking.
```

Comment:
if u find somewhere the grammer rule send me a link.. i want to know this comma rules and it seems i have missed it in school